### DISCUSSION TITLE
Support Subsonic clients/players on mobile phone

### DISCUSSION THREAD

**Discussion by xiaolei0125** [music-assistant/support#2869] [Feature requests and ideas]
Does music-assistant plan to support Subsonic API?   
   So I can use a Subsonic clients player(Android APP: Symfonium/DSub) to play music from music-assistant.
   Then music-assistant can replace [navidrome](https://www.navidrome.org/docs/overview/#apps)

**Comment by xiaolei0125** [comment]
Related requests:
	- [Play Music on "current" Device/Browser #2347](https://github.com/orgs/music-assistant/discussions/2347)
	- [Phone app #1834](https://github.com/orgs/music-assistant/discussions/1834)

**Reply by OzGav** [reply]
Unlikely. The Snapcast app works on mobile now and after that HA Companion apps would be next

**Comment by Hedda** [comment]
For reference, "Navidome" is a competing self-hosted music-server that also uses Subsonic API and relies on Subsonic client ecosystem:

- https://www.navidrome.org

  * https://www.navidrome.org/about/

     * https://www.navidrome.org/docs/overview/#apps

       * https://github.com/navidrome/navidrome

Navidrome is an open-source web-based music server and streaming platform where you can store your own music and listen to it on multiple clients. Deployment ...

**Comment by Hedda** [comment]
For reference; here are some of the Airsonic and Subsonic clients for different platforms (note that [Airsonic ](https://airsonic.github.io/)and [Subsonic](http://www.subsonic.org/) provide the same API):

* https://airsonic.github.io/docs/apps/

### Android 

- [Symfonium](https://symfonium.app/)
- [DSub](https://play.google.com/store/apps/details?id=github.daneren2005.dsub)
- [StreamMusic](https://music.aqzscn.cn/docs/versions/latest/)
- [substreamer](https://substreamerapp.com/)
- [Tempo](https://github.com/CappielloAntonio/tempo#readme)
- [Subtracks](https://github.com/austinried/subtracks#readme)
- [Ultrasonic](https://ultrasonic.gitlab.io/)

### iOS

- [play:Sub](http://michaelsapps.dk/playsubapp/)
- [StreamMusic](https://apps.apple.com/ca/app/stream-music-enjoy-music/id6449966496)
- [substreamer](https://substreamerapp.com/)
- [Amperfy](https://github.com/BLeeEZ/amperfy#readme)
- [flo](https://client.flooo.club/)
- [iSub](https://isub.app/)

### Web

- [Feishin](https://feishin.vercel.app/)
- [Thunderdrome](https://thunderdrome.netlify.app/)
- [Airsonic Refix](https://airsonic.netlify.com/)
- [Aonsoku](https://aonsoku.vercel.app/)
- [Subplayer](https://subplayer.netlify.app/)
- [Aurial](https://shrimpza.github.io/aurial/)
- [Subfire](https://p.subfireplayer.net/)
- [Jamstash](https://tsquillario.github.io/Jamstash)

### Desktop

GUI:
- [Feishin](https://github.com/jeffvli/feishin)   
- [Supersonic](https://github.com/dweymouth/supersonic)   
- [Submariner](https://submarinerapp.com/) 
- [StreamMusic](https://music.aqzscn.cn/docs/versions/latest/)  
- [Aonsoku](https://github.com/victoralvesf/aonsoku)   
- [Amperfy](https://github.com/BLeeEZ/amperfy#readme) 
- [Sonixd](https://github.com/jeffvli/sonixd)   
- [Sublime Music](https://sublimemusic.app/) 

CLI:
- [STMP](https://github.com/wildeyedskies/stmp#readme)  
- [STMPS](https://github.com/spezifisch/stmps#readme)  
- [termsonic](https://git.sixfoisneuf.fr/termsonic/about/) 
- [Jellycli](https://github.com/tryffel/jellycli#readme)

### Other

Apple TV:
  - [SubSwift](https://apps.apple.com/us/app/subswift/id6504658929)

Connected Speakers:
  - Sonos: [bonob](https://github.com/simojenki/bonob#readme)
 
  - Alexa: [AskSonic](https://github.com/srichter/asksonic#readme)

Other:
  - [Subsonic Kodi Plugin](https://github.com/warwickh/plugin.audio.subsonic#readme)
  - [Navidrome Kodi Plugin](https://github.com/BobHasNoSoul/plugin.audio.navidrome#readme)
  - [HTTPDirFS](https://github.com/fangfufu/httpdirfs#readme)
  - [upmpdcli](https://www.lesbonscomptes.com/upmpdcli/index.html): expose Navidrome as a UPnP/DLNA media library. See the [discussion](https://github.com/navidrome/navidrome/discussions/2324).

**Comment by bfranske** [comment]
Came here for this. I'd like to find a unified solution for playing at home (on players) and while out and about (on phones and android auto/carplay). For playback on phones I am currently using Navidrome with Subsonic phone clients because I do sometimes travel in areas with poor or no phone service so the ability to cache songs locally is important. It seems like the easiest path would be to just support Subsonic clients from Music Assistant.

FWIW I want to unify my music playback solution so that playlists can be synchronized (i.e. I can build a playlist at home for use on a player but also play the same playlist on a mobile device).
